app/assets/javascripts/index.js

```diff
@@ -1,5 +1,9 @@
 module.exports.Result = require('./Result.js').Result;
 module.exports.ResultSchema = require('./Result.js').ResultSchema;
+module.exports.Measure = require('./Measure.js').Measure;
+module.exports.MeasureSchema = require('./Measure.js').MeasureSchema;
+module.exports.MeasurePackage = require('./MeasurePackage.js').MeasurePackage;
+module.exports.MeasurePackageSchema = require('./MeasurePackage.js').MeasurePackageSchema;
```

Comment:
These lines shouldn't be added manually. Since this file is regenerated each time, you'll need to put these lines in the regeneration script.

Reply:
Good catch, thanks